lib/packages.js

```diff
@@ -0,0 +1,223 @@
+'use strict';
+
+// requires
+var Log = require('log');
+var testing = require('testing');
+var db = require('./db.js');
+var config = require('../config.js');
+
+// globals
+var packagesCollection;
+var log = new Log(config.logLevel);
+
+
+function openCollection(callback)
+{
+	if (packagesCollection)
+	{
+		return callback(null, packagesCollection);
+	}
+	db.addCallback(function(error, result)
+	{
+		if (error)
+		{
+			return callback('Could not access database: ' + error);
+		}
+		packagesCollection = result.collection(config.packagesCollection);
+		if (packagesCollection)
+		{
+			console.log('was there');
```

Comment:
I was just debugging an issue with tests, done in commit 9600516cc052617a69b5d550639623149246a49f.